content/applications/productivity/discuss/advanced/email_servers.rst

```diff
@@ -136,20 +136,38 @@ Instead, you can also configure Odoo to do something similar by itself:
     .. image:: media/alias_domain.png
        :align: center
 
+#.  Click on *Outgoing Mail Servers*
+
+#.  Create a new one.
+
+#.  Fill its *From Filter*.
+
+    .. image:: media/outgoing_server_from_filter.png
+       :align: center
+
+    * Use a domain (such as ``mycompany.example.com``) to keep the original "From"
+      address for mails that come from that domain.
+
+    * Use an address (such as ``outgoing@mycompany.example.com``) to allow only
+      that outgoing address.
+      
+    * Keep it empty to use this server for any email address.
+
+With this configuration in place, if Odoo sends an email that doesn't match any
+of the *from filters*, it will alter the email's "From" before sending it to
+the MTA.
+
+It will use the default outgoing email address, composed like this:
+``{mail.default.from}@{mail.catchall.domain}``.
```

Comment:
Indent this block to avoid breaking the numbering.
```suggestion
    With this configuration in place, if Odoo sends an email that doesn't match any
    of the *from filters*, it will alter the email's "From" before sending it to
    the MTA.

    It will use the default outgoing email address, composed like this:
    ``{mail.default.from}@{mail.catchall.domain}``.
```